LTTngDataExtensions/DataOutputTypes/LTTngGenericEvent.cs

```diff
@@ -25,6 +27,20 @@ public EventKind(uint id, string name, IReadOnlyList<CtfFieldValue> fields)
                 this.FieldNames.Add(field.FieldName);
             }
         }
+
+        private static readonly Dictionary<Tuple<string, uint>, EventKind> RegisteredKinds = new Dictionary<Tuple<string, uint>, EventKind>();
```

Comment:
NIT: I am not a big fan of Tuples at all. I would much rather we use a class. It's better for readability & refactoring

Reply:
Can do. My rationale for the tuple was that it provides dictionary support (Equals, GetHashCode) for free.

Reply:
makes sense - thx though for implementing those